drivers/ipm/Kconfig

```diff
@@ -22,3 +22,10 @@ config IPM_QUARK_SE_MASTER
 	  Sets up the initial interrupt mask and clears out all
 	  channels. Should be turned on for one CPU only.
 
+config IPM_MCUX
```

Comment:
Should this be IPM_MCUX_LPC (fixup below as well if we make this LPC specific)

Reply:
This is for all SoC's which have HAS_MCUX defined: LPC, Kinetis, i.MX. So it is not only for LPC.